packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3489,6 +3489,27 @@ export default class Meeting extends StatelessWebexPlugin {
     await this.publishStream(MediaType.AudioSlides, this.mediaProperties.shareAudioStream);
   }
 
+  /**
+   * Handles the local audio stream publish state change event
+   * @internal
+   * @returns {undefined}
+   */
+  private emitPublishStateChangeEvent({functionName, isPublished, mediaType, stream}) {
```

Comment:
types are missing for all the parameters

Reply:
Done

Reply:
mediaType should be MediaType not string

Reply:
Sorry. Thanks for noticing. Done.